backend/src/apiserver/model/job.go

```diff
@@ -23,6 +23,7 @@ type Job struct {
 	Namespace          string `gorm:"column:Namespace; not null;"`
 	Description        string `gorm:"column:Description; not null"`
 	MaxConcurrency     int64  `gorm:"column:MaxConcurrency;not null"`
+	NoCatchup          bool   `gorm:"column:NoCatchup; not null"`
```

Comment:
Just want to double check on how mysql handles this schema change. When the new column is added, the existing rows don't have value for this column. Since the column is said to be not null, what would be the default value be used or upgrade doesn't go through?

Reply:
That's a good point. I think default value will be false, but I'm not sure, let me double check on a real db.

Reply:
```
mysql> select Name, NoCatchup from jobs;
+-----------------------------+-----------+
| Name                        | NoCatchup |
+-----------------------------+-----------+
| periodic-catchup-falsehtrd9 |         0 |
| cron-catchup-true6km84      |         0 |
| periodic-catchup-truezcwhs  |         0 |
| periodic-catchup-falsefd272 |         0 |
| catchupfalsesslv8           |         0 |
| cron-catchup-falsemx58x     |         0 |
| periodic-catchup-true-znwf8 |         0 |
| catchuptruec9vr6            |         0 |
| periodic-catchup-truekdq5d  |         0 |
| cron-catchup-falsefcpdl     |         0 |
| periodic-catchup-truenn4qr  |         0 |
| cron-catchup-true6qxdq      |         0 |
+-----------------------------+-----------+
12 rows in set (0.00 sec)
```
Looks like it's correct, this column will be auto populated with false, so it matches backward compatible behavior.